FredMadrona/nextJS-HardwaresPH
# 4 customerpassword reset forgot

### 🎫 Ticket

- 4 customerpassword reset forgot

### 📦 Packages

- none

### 📝 Updates

- added  a link from login page to forgot password page via forgot password text
- added the forgot password page
- added the image slider in the homepage
- added the logout button in the navbar

### 🧪 Testing Steps

- List the testing steps for this update.
- click the forgot password text
- type email and submit if it posts the reset prompt
- navigate to pages if the username is displayed
![forgotpassword](https://github.com/FredMadrona/nextJS-HardwaresPH/assets/79515515/17f58d8b-54ad-4fd1-84ad-ec84d220d020)
